src/modules/display.js

```diff
@@ -1,12 +1,16 @@
-import scores from './scores';
-
 const display = () => {
-  const scoresList = document.getElementById('scoresList');
-  scoresList.innerHTML = '';
-  scores.forEach((score) => {
-    scoresList.innerHTML += `
-    <li>${score.Name}: ${score.Score}</li>
-    `;
+  const refresh = document.getElementById('refresh');
+  refresh.addEventListener('click', async () => {
+    const scoresList = document.getElementById('scoresList');
+    scoresList.innerHTML = '';
+    let scores = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Jun0BzNiB6z5IoA83KCo/scores/');
+    scores = await scores.json();
+    scores = scores.result;
+    scores.forEach((score) => {
+      scoresList.innerHTML += `
+      <li>${score.user}: ${score.score}</li>
+      `;
+    });
   });
 };
```

Comment:
Hey @Anand8317, great job with the application. However, there is this one issue that you need to fix. I have attached [this video here](https://www.loom.com/share/64f7e8dcce324c92b79050fb214589e7), you can go through it where I explained the issue in detail.

Reply:
Thanks for explaining the issue in detail. Highly appreciate the help. I have fixed the rendering on the page load issue. Again, thanks.